documentation/rpi3.md

```diff
@@ -1,19 +1,17 @@
 # Raspberry Pi 3 on OP-TEE
-[Sequitur Labs](http://www.sequiturlabs.com) did the initial port which besides
-the actual OP-TEE port also patched U-boot, ARM Trusted Firmware and Linux
-kernel. Sequitur Labs also pulled together patches for OpenOCD to be able to
-debug the solution using cheap JTAG debuggers. For more information about the
-work, please see the [press
-release](http://www.sequiturlabs.com/media_portfolio/sequitur-labs-collaborates-with-linaro-to-lower-barriers-to-iot-security-education-for-raspberry-pi-maker-community)
-from June 8 2016.
+[Sequitur Labs] did the initial port which besides the actual OP-TEE port also
+patched U-boot, ARM Trusted Firmware and Linux kernel. Sequitur Labs also pulled
+together patches for OpenOCD to be able to debug the solution using cheap JTAG
+debuggers. For more information about the work, please see the [press
+release] from June 8 2016.
```

Comment:
That should be fine I think, as the commit message says, I've moved most of the links to the bottom of the page now. Have look at the rendered version and you should see a real link I guess (and hope).